docs/pages/inboxes/create-conversations.md

```diff
@@ -133,6 +133,47 @@ let group = try await alix.conversations.newGroup([bo.inboxId, caro.inboxId],
 
 :::
 
+## Optimistically create a group chat
+
+Use this method to optimistically create a group chat, which enables a user to create a group chat now and add members later.
+
+The group chat is created with just a name and is stored only in the app's local storage. The group chat is visible only to the creator in the app installation they used to create it.
+
+When you want to add members, you use [`addMembers()`](/inboxes/group-permissions#add-members-by-inbox-id) with a list of inbox IDs.
+
+After adding members, you need to call [`sync()`](/inboxes/sync-and-syncall#sync-a-specific-conversation) to synchronize the group chat with the network.
```

Comment:
you shouldn't need to call sync here. the group syncs automatically when adding or removing members.